test/e2e/e2e.go

```diff
@@ -175,6 +175,59 @@ var _ = Describe("validation", func() {
 			})
 		})
 	})
+
+	Context("MetalLB contains incorrect data", func() {
+		Context("MetalLB has incorrect name", func() {
+
+			var metallb *metallbv1alpha.Metallb
+			var metallbCRExisted bool
+
+			BeforeEach(func() {
+				metallb = &metallbv1alpha.Metallb{}
+				err := loadMetallbFromFile(metallb, MetallbCRFile)
+				Expect(err).ToNot(HaveOccurred())
+				metallb.SetName("incorrectname")
+				Expect(testclient.Client.Create(context.Background(), metallb)).Should(Succeed())
+			})
+
+			AfterEach(func() {
+				if !metallbCRExisted {
```

Comment:
No need for this flag anymore

Reply:
If you remove it, just go out if the err is not found in the following line

Reply:
indeed, done